### PR TITLE
Toyota: Lexus IS 2023 port

### LINKED ISSUE #29254: Lexus IS 2023 [vehicle is not recognized and is in dashcam mode]
### Describe the bug

Installed Comma 3 with Toyota harness into 2023 Lexus IS, vehicle is not recognized.

### Which car does this affect?

Lexus IS 2023

### Provide a route where the issue occurs

649bf2997ada6e3a|2023-08-04--22-20-47

### openpilot version

0.9.5

### Additional info

Attempted to fingerprint myself but I could only get as far as "openpilot unavailable; waiting for controls to start."

### PR DESCRIPTION
**Pull request by L3R5**
I am using a Comma 3 with a 2023 Lexus IS

With normal openpilot installed, the screen displays "car unrecognized: dashcam mode"
route with normal openpilot: 649bf2997ada6e3a|2023-08-04--22-20-47

With the addition of CAR.LEXUS_IS_TSS2, the screen displays "openpilot unavailable, waiting for controls to start"
route with my attempt at fingerprinting: 649bf2997ada6e3a|2023-08-06--06-21-58

Turning on/off cruise control or lane trace assist does not change "openpilot unavailable"

### CODE REVIEW

**Comment by L3R5** [comment]
Update: upon seeing "NotImplementedError: Did not find torque params for LEXUS IS 2023" in the PR checks, I tried using the same torque params as LEXUS IS 2018 using torque_data/substitute.yaml

After doing that, the "openpilot unavailable" message has gone away and I now see a "CAN error: check connections" message which persists even after disconnecting and reconnecting all of the cables
relevant route: 649bf2997ada6e3a|2023-08-06--02-56-17

File: selfdrive/car/toyota/values.py [hunk 1]
```diff
@@ -2276,11 +2296,11 @@ class ToyotaCarInfo(CarInfo):
 }
 
 # These cars have non-standard EPS torque scale factors. All others are 73
-EPS_SCALE = defaultdict(lambda: 73, {CAR.PRIUS: 66, CAR.COROLLA: 88, CAR.LEXUS_IS: 77, CAR.LEXUS_RC: 77, CAR.LEXUS_CTH: 100, CAR.PRIUS_V: 100})
+EPS_SCALE = defaultdict(lambda: 73, {CAR.PRIUS: 66, CAR.COROLLA: 88, CAR.LEXUS_IS: 77, CAR.LEXUS_IS_TSS2: 77, CAR.LEXUS_RC: 77, CAR.LEXUS_CTH: 100, CAR.PRIUS_V: 100})
```
Comment (sshane) [inline selfdrive/car/toyota/values.py:2299]:
```suggestion
EPS_SCALE = defaultdict(lambda: 73, {CAR.PRIUS: 66, CAR.COROLLA: 88, CAR.LEXUS_IS: 77, CAR.LEXUS_RC: 77, CAR.LEXUS_CTH: 100, CAR.PRIUS_V: 100})
```

Comment (sshane) [inline selfdrive/car/toyota/values.py:2299]:
```suggestion
EPS_SCALE = defaultdict(lambda: 73, {CAR.PRIUS: 66, CAR.COROLLA: 88, CAR.LEXUS_IS: 77, CAR.LEXUS_RC: 77, CAR.LEXUS_CTH: 100, CAR.PRIUS_V: 100})
```

File: selfdrive/car/toyota/values.py [hunk 2]
```diff
@@ -2289,7 +2309,7 @@ class ToyotaCarInfo(CarInfo):
 UNSUPPORTED_DSU_CAR = {CAR.LEXUS_IS, CAR.LEXUS_RC}
 
 # these cars have a radar which sends ACC messages instead of the camera
-RADAR_ACC_CAR = {CAR.RAV4H_TSS2_2022, CAR.RAV4_TSS2_2022, CAR.RAV4H_TSS2_2023, CAR.RAV4_TSS2_2023, CAR.CHR_TSS2, CAR.CHRH_TSS2}
+RADAR_ACC_CAR = {CAR.RAV4H_TSS2_2022, CAR.RAV4_TSS2_2022, CAR.RAV4H_TSS2_2023, CAR.RAV4_TSS2_2023, CAR.CHR_TSS2, CAR.CHRH_TSS2, CAR.LEXUS_IS_TSS2}
```
Comment (sshane) [inline selfdrive/car/toyota/values.py:2312]:
```suggestion
RADAR_ACC_CAR = {CAR.RAV4H_TSS2_2022, CAR.RAV4_TSS2_2022, CAR.RAV4H_TSS2_2023, CAR.RAV4_TSS2_2023, CAR.CHR_TSS2, CAR.CHRH_TSS2}
```

File: selfdrive/car/toyota/values.py [hunk 3]
```diff
@@ -2261,6 +2280,7 @@ class ToyotaCarInfo(CarInfo):
   CAR.LEXUS_ESH: dbc_dict('toyota_new_mc_pt_generated', 'toyota_adas'),
   CAR.SIENNA: dbc_dict('toyota_tnga_k_pt_generated', 'toyota_adas'),
   CAR.LEXUS_IS: dbc_dict('toyota_tnga_k_pt_generated', 'toyota_adas'),
+  CAR.LEXUS_IS_TSS2: dbc_dict('toyota_nodsu_pt_generated', 'toyota_tss2_adas'), # not certain if this is correct
```
Comment (sshane) [inline selfdrive/car/toyota/values.py:2283]:
```suggestion
  CAR.LEXUS_IS_TSS2: dbc_dict('toyota_nodsu_pt_generated', 'toyota_tss2_adas'),
```

**Pull request merged**